docs/ref/csrf.txt

```diff
@@ -146,6 +148,16 @@ If you're using AngularJS 1.1.3 and newer, it's sufficient to configure the
     $httpProvider.defaults.xsrfCookieName = 'csrftoken';
     $httpProvider.defaults.xsrfHeaderName = 'X-CSRFToken';
 
+CSRF token in Jinja2 templates
+------------------------------
+
+The Jinja2 template language exposes ``{{ csrf_input }}`` which is equivalent
```

Comment:
This is a nit-pick, but I think accuracy is worthwhile here: the Jinja2 template language doesn't expose `{{ csrf_input }}`, rather Django's Jinja2 template engine exposes it. (That is to say, you won't find any reference to `csrf_input` anywhere in the Jinja2 docs, it's added to the template context by Django's Jinja2-wrapper engine class.) So I'd phrase this sentence as "Django's Jinja2 template engine exposes..."